Entwicklerdokumentation.tex

```diff
@@ -3,5 +3,6 @@
 \usepackage[ngerman]{babel}
 
 \begin{document}
+  \input{Gespraeche}
```

Comment:
Kommen die Gespräche nochmal in die Entwicklerdoku? Die sind doch schon in der Projektdoku...